Vagrantfile

```diff
@@ -4,7 +4,7 @@
 build_boxes = [
   'appimage',
   'archlinux',
-  'centos-7',
+  'centos-6',
```

Comment:
is this meant to be building centos-6? It [eol'ed in 2020](https://cloud.google.com/compute/docs/eol/centos6)

Reply:
yes, mainly for redhat 6 as it is still in extended support